components/text/hunspell/Makefile

```diff
@@ -41,6 +42,13 @@ CPPFLAGS += -I/usr/include/ncurses
 CFLAGS += -D_XOPEN_SOURCE=600
 CFLAGS += -std=c99
 
+CPPFLAGS +=	-I/usr/include/ncurses
+
+# Check location of ncurses libraries
+ifeq ($(shell test -f /usr/gnu/lib/libncurses.so && echo y),y)
```

Comment:
There's no hacking of flags.  That's part of the design of the oi-userland build system.  Many macros are defined in make-rules/shared-macros.mk so that they can be added to compile flags in the component Makefiles.  About 10% of them take advantage of this feature.  Yes, pkg-config would be useful, but only if it were correct, and only if it handled 32 and 54-bit builds.